File: advertiser/advertiser/Models/Account.cs
namespace advertiser
{
	public class Account
	{
		public Account (int id, string name, string password)
		{
			this.id = id;
			this.name = name;
			this.password = password;
		}
		

		private int id;
		public int Id
		{
			get { return Id; }
		}

		private string name;
		public string Name
		{
			get { return name; }
		}
		
		private string password;
		public string Password
		{
			get { return password; }
		}

	}
}

File: advertiser/public/javascripts/account.js
// account.js

function account_create()
{
    name = $('new_account')['account_name'].getValue();
    email = $('new_account')['account_email'].getValue();
    password = $('new_account')['account_password'].getValue();
    
    var account = {'name': name, 'email' : email, 'password' : password.md5() }; 
   
    new Ajax.Request("accounts", {
	method: 'post',
	parameters: account ,
	onSuccess: function(transport) {
	    console.log("accounts : post returns " + transport);
	}
    });
 
}

function new_onload()
{
    $('account_name').focus();
}

